pkgs/by-name/rc/rcp/package.nix

```
@@ -0,0 +1,34 @@
+{ lib, stdenv
+, buildPackages
+, fetchFromGitHub
+, rustPlatform
+, pkg-config
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rcp";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "wykurz";
+    repo = pname;
```

Nit (as this only really matters when using `stdenv.mkDerivation` with the `finalAttribute` pattern):
```suggestion
    repo = "rcp";
```

---

Please don't use this.

`repo = pname;` is nice for DRY but creates a binding that goes too far.

See further information about this here: https://github.com/nix-community/nixpkgs-lint/issues/21